core/src/main/scala/geotrellis/server/extent/SampleUtils.scala

```diff
@@ -48,6 +43,20 @@ object SampleUtils {
     (tl, tr, bl, br)
   }
 
+  /** Choose the largest cellsize with the minCells amount */
+  final def chooseLargestCellSize(rasterExtents: NEL[RasterExtent], minCells: Int): CellSize =
+    rasterExtents
+      .reduceLeft { (chosenRE: RasterExtent, nextRE: RasterExtent) =>
+        val (chosenCS, nextCS) = chosenRE.cellSize -> nextRE.cellSize
+        val chosenSize = chosenCS.height * chosenCS.width
+        val nextSize = nextCS.height * nextCS.width
+
+        if (nextSize > chosenSize && nextRE.size > minCells)
```

Comment:
@CloudNiner `feature/stac-support` is out of date, this code is from https://github.com/geotrellis/geotrellis-server/pull/261

Reply:
Ah didn't realize that was from a previous PR. In any case, my question stands since it's about the implementation appearing to do something incorrect at first glance. If this is a problem, perhaps a separate issue if it can't be resolved now?

Reply:
@CloudNiner :D it is already in develop https://github.com/geotrellis/geotrellis-server/blob/develop/core/src/main/scala/geotrellis/server/extent/SampleUtils.scala 

The `feature/stac-support` branch is behind the `develop` branch. Don't worry about this piece of code, it would go after rebasing on develop.

Reply:
Ah ok. Roger.